apolline-flutter/lib/widgets/charts/BatteryLevelIndicator.dart

```diff
@@ -7,7 +7,7 @@ class BatteryLevelIndicator extends StatefulWidget {
   final double currentBatteryLevel;
   final double minimumLevel = 0;
   final double maximumLevel = 100;
-  final double minimumSensorBatteryLevel = 3.50;
+  final double minimumSensorBatteryLevel = 2.90;
```

Comment:
Current battery level should be computed with following data:

![perc](https://user-images.githubusercontent.com/11993538/134707177-1a7e4767-0673-4be7-b66f-5e3564bb101d.png)

Reply:
Implemented in e2cd62d0c8d260f55aca86309319f2b9d01cc665.